CompetitionTemplate/src/competition/operator_interface/OperatorInterface.java

```diff
@@ -34,10 +37,13 @@ public OperatorInterface(WPIFactory factory, RobotAssertionManager assertionMana
         rightJoystick.setYInversion(true);
         operatorJoystick.setXInversion(true);
         operatorJoystick.setYInversion(true);
+        operatorPanel.setXInversion(true);
```

Comment:
Also no reason to do this inversion at all, we won't have any joystick axis on the panel right now